cftemplates/snapshots_tool_rds_source.json

```diff
@@ -28,10 +28,10 @@
 			"Default": "24",
 			"Description": "Interval for backups in hours. Default is 24"
 		},
-		"DestinationAccount": {
-			"Type": "Number",

```

Comment:
I feel like this was never correct. Not sure if AWS account IDs have leading zeros but, if so, this would have broke for those 🙃